vnext/src/Libraries/Modules/Alert/Alert.windows.ts

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ * @format
+ */
+'use strict';
+
+import {NativeModules} from 'react-native';
```

Comment:
Please file an issue to move this code into react-native, and add a TODO here tracking it